src/main/java/net/imglib2/algorithm/morphology/table2d/Spur.java

```diff
@@ -73,7 +73,7 @@
 
 		final ImgFactory< T > factory;
 		if ( extendedVal instanceof NativeType )
-			factory = ( ImgFactory< T > ) Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
+			factory = Util.< NativeType >getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```

Comment:
@tpietzsch @hanslovsky both java8 and eclipse are stupid, I can infer what's going on and eventually it doesn't make a difference anyways.  But who am I to judge compiler design ;).  @hanslovsky's version would save an explicit runtime cast but eclipse not accepting it is a red flag, so I reverted it:

https://github.com/imglib/imglib2-algorithm/commit/b7bdb933ea42cc1fb233e3006f341f05c5897824